SampleComponentLibraryApp/SampleComponentLibraryApp/NavigationHeaderView/NavigationHeaderViewViewController.swift

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+class NavigationHeaderViewViewController: UIViewController, Themeable {
```

Comment:
Since the code surrounding this is very similar to HeaderView, what do you think about creating one VC for both? Similar to how we do our `Buttons`, we can call this one `Headers` and have the 3 different views on one page. Can also add a title on top of the views for separation.